Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12-stretch
+FROM golang:1.14-stretch
```

Comment:
The default should be still kept <= 1.13. But you can override the version to 1.14 with `--build-arg` in CI.